libs/utils/include/celix_errno.h

```diff
@@ -178,6 +178,7 @@ CELIX_UTILS_EXPORT const char* celix_strerror(celix_status_t status);
  */
 
 #define CELIX_ENOMEM                    CELIX_ERROR_MAKE(CELIX_FACILITY_CERRNO,ENOMEM)
+#define CELIX_EAGAIN                    CELIX_ERROR_MAKE(CELIX_FACILITY_CERRNO,EAGAIN)
```

Comment:
If we make it explicit that `celix_status_t` contains all errno, then this is unnecessary.
Of course we should also extend `celix_strerror`.

Reply:
Agree, I will update this